fortuna/src/casus/blocks/SetVariableBlock.js

```diff
@@ -0,0 +1,92 @@
+//@flow strict
+
+import BoundingBox from './BoundingBox.js';
+import CasusBlock from './CasusBlock.js';
+import EmptyBlock from './EmptyBlock.js';
+import Vec from './Vec.js'
+import generateCornerPerim from './generateCornerPerim.js'
```

Comment:
Again, warning that the fields aren't used. Again, if you'll use them eventually, keep, if not, remove.

Reply:
Fixed :)